perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -37,4 +40,27 @@ sub reopening_disallowed {
     return 1;
 }
 
+sub open311_extra_data_include {
+    my ($self, $row, $h) = @_;
+
+    my $open311_only = [
+        { name => 'report_url',
+          value => $h->{url} },
+        { name => 'title',
+          value => $row->title },
+        { name => 'description',
+          value => $row->detail },
+        { name => 'category',
+          value => $row->category },
+    ];
+
+    return $open311_only;
+}
+
+sub title_fetched_report {
```

Comment:
I think it would be good to prefix this method with `open311_` to match the other hooks that are specific to open311 and to give a hint as to where this hook is called.